internal/pkg/manifest/rd_web_svc.go

```diff
@@ -97,7 +101,10 @@ func (s *RequestDrivenWebService) TaskPlatform() (*string, error) {
 	if s.InstanceConfig.Platform == nil {
 		return nil, nil
 	}
-	return s.InstanceConfig.Platform.PlatformString, nil
+	if isWindowsPlatform(s.InstanceConfig.Platform) {
+		return nil, errors.New("Windows is not supported for App Runner services")
+	}
```

Comment:
Should this validation be placed in `windowsCompatibility()` instead?

Reply:
Good point! Moved.